uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/models/instances/leave-request-instance_test.js

```diff
@@ -575,5 +575,25 @@ define([
         return $q.resolve(value);
       }
     });
+
+    describe('toAPI()', function () {
```

Comment:
LeaveRequestInstance does not have a `toAPI` method. This test case should not be here